fuse_optimizers/src/batch_optimizer.cpp

```diff
@@ -184,16 +188,21 @@ void BatchOptimizer::optimizationLoop()
       const_transaction = std::move(combined_transaction_);
       combined_transaction_ = fuse_core::Transaction::make_shared();
     }
-    // Update the graph
-    graph_->update(*const_transaction);
-    // Optimize the entire graph
-    graph_->optimize();
-    // Make a copy of the graph to share
-    fuse_core::Graph::ConstSharedPtr const_graph = graph_->clone();
-    // Optimization is complete. Notify all the things about the graph changes.
-    notify(const_transaction, const_graph);
-    // Clear the request flag now that this optimization cycle is complete
-    optimization_request_ = false;
+
+    {
+      TRACE_SCOPE_RAII("Optimize.");
```

Comment:
I would remove the dot at the end, just make it `Optimize`, there's one down below as well. But it's a matter of taste.

Reply:
I saw some code doing that, but I actually prefer it w/o the trailing dot. :smiley:

Reply:
Changed